dashboard/src/types/issueExtras.ts

```diff
@@ -0,0 +1,26 @@
+import type { TreeBranchItem } from './general';
+import type { IssueKeys } from './issues';
+
+enum PossibleIssueTags {
+  Mainline = 'mainline',
+  Stable = 'stable',
+  LinuxNext = 'linux-next',
+}
```

Comment:
https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh

```suggestion
const PossibleIssueTags = {
  mainline: 'mainline',
  stable: 'stable',
  linuxNext : 'linux-next',
} as const
```